modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/entries/display/context/ObjectEntryDisplayContext.java

```diff
@@ -737,26 +737,47 @@ private boolean _isActive(ObjectField objectField) throws PortalException {
 		return true;
 	}
 
+	private void _removeTimeFromDateString(
+		Map<String, Serializable> values, DDMFormField ddmFormField) {
+
+		for (Map.Entry<String, Serializable> entry : values.entrySet()) {
+			if (!Objects.equals(entry.getKey(), ddmFormField.getName())) {
+				continue;
+			}
+
+			String dateString = String.valueOf(entry.getValue());
+
+			values.replace(
+				entry.getKey(), entry.getValue(),
+				dateString.replaceAll(
+					" [0-9]{1,2}:[0-9]{1,2}:[0-9]{1,2}.[0-9]", ""));
+		}
+	}
+
 	private void _setDateDDMFormFieldValue(
 		List<DDMFormField> ddmFormFields, Map<String, Serializable> values) {
 
 		for (DDMFormField ddmFormField : ddmFormFields) {
-			if (!StringUtil.equals(ddmFormField.getType(), "date")) {
-				continue;
-			}
+			if (StringUtil.equals(ddmFormField.getType(), "fieldset")) {
+				List<DDMFormField> nestedDDMFormFields =
+					ddmFormField.getNestedDDMFormFields();
 
-			for (Map.Entry<String, Serializable> entry : values.entrySet()) {
-				if (!Objects.equals(entry.getKey(), ddmFormField.getName())) {
-					continue;
-				}
+				for (DDMFormField nestedDDMFormField : nestedDDMFormFields) {
+					if (!StringUtil.equals(
```

Comment:
Hey @danvb112, you could compare the string without denying the result. I mean, you could call the `_removeTimeFromDateString` method inside the `if` clause.

Reply:
There are many situations that Brian prefers the "denied condition" + continue

Reply:
I see, but in this case, Brian made a "denied condition" + "continue" because the code was used only in one place. I think that is why he adopts the "buy space" strategy. But with a method creation, I think it is not necessary the "buy space" strategy anymore.